lib/tasks/scripts/import_facility_block_names.rb

```diff
@@ -0,0 +1,22 @@
+class ImportFacilityBlockNames
+  def self.import(file)
+    facilities_updated = 0
+    facilities_not_found = 0
+
+    CSV.foreach(file, headers: true) do |row|
+      district = row["district"]
+      facility_name = row["facility"]
+      block = row["block"]
+
+      facility = Facility.where(name: facility_name, district: district)
```

Comment:
`find_by`? Do we expect more than one?

Reply:
There's a bunch of dupe facilities in prod and it's hard to tell which is the "correct" one. Just another bad data edge case 😅 .